namui/src/namui/common/types/simple_signed.rs

```diff
@@ -0,0 +1,18 @@
+use num::Signed;
+
+/// It is similar to `num::Signed`,
+/// but it implements only `is_positive` and `is_negative`.
+pub trait SimpleSigned {
+    fn is_sign_positive(&self) -> bool;
+    fn is_sign_negative(&self) -> bool;
+}
```

Comment:
I implemented new trait `SimpleSigned` instead of `num::Signed`.
`num::Signed` requires [Num](https://docs.rs/num/latest/num/trait.Num.html) + [Neg](https://doc.rust-lang.org/nightly/core/ops/arith/trait.Neg.html)<Output = Self>.
I think that is overhead. So I made it.

Code below is about compatibility for `num::Signed`.